packages/viz/src/Chart/Chart.d.ts

```diff
@@ -4,10 +4,22 @@ import { HvPlotProps } from "./Plot";
 export type HvChartClassKey = "root";
 
 export interface HvChartProps extends StandardProps<HvPlotProps, HvChartClassKey> {
+  /**
+   * An Id passed on to the component
+   */
+  id?: string;
   /**
    * Defines if should use a single or multiline tooltip.
    */
   tooltipType?: "single" | "multiple";
+  /**
+   *
+   */
+  customTooltip?: React.ReactNode;
+  /**
+   * Custom object containing any value to send to the customTooltip
+   */
+  tooltipData?: Record<string, unknown>;
```

Comment:
Greetings @rubenbarroshv thank you for your contribution, to be able to merge we have 3 points for you to improve

1- Missing documentation in line 15 - 17,
2- we could shorten the name `customTooltip` to only tooltip because there is no other prop using that name, just specify in the documentation that it is an override
3- maybe we could eliminate the `tooltipData `Prop and just pass the data embedded in the component, unless the prop needs to be somehow mapped with an internal value of the chart

Reply:
I was trying to use the tooltipData before, but you are right, I don't need it anymore. If I will need it again I will propose another update 😄